examples/sn-dms-demo/src/components/ActionMenu/ActionMenu.tsx

```diff
@@ -311,7 +319,9 @@ class ActionMenu extends React.Component<
           this.props.history.replace(`/documents?query=${query.Query}&queryName=${query.DisplayName || query.Name}`)
           this.props.closeActionMenu()
           break
-
+        case 'OpenInEditor':
+          return null
+          break
```

Comment:
```suggestion
```
break is not needed if there is a return.

Reply:
Also IMO it would be better to return early in line 173 like this:
```js
      if (!content || action.Name === 'OpenInEditor') {
        return
      }
```